ci/docker/python-wheel-windows-vs2019.dockerfile

```
@@ -84,7 +84,7 @@ RUN (if "%python%"=="3.9" setx PYTHON_VERSION "3.9.13" && setx PATH "%PATH%;C:\P
     (if "%python%"=="3.10" setx PYTHON_VERSION "3.10.11" && setx PATH "%PATH%;C:\Python310;C:\Python310\Scripts") & \
     (if "%python%"=="3.11" setx PYTHON_VERSION "3.11.9" && setx PATH "%PATH%;C:\Python311;C:\Python311\Scripts") & \
     (if "%python%"=="3.12" setx PYTHON_VERSION "3.12.5" && setx PATH "%PATH%;C:\Python312;C:\Python312\Scripts") & \
```

Not strictly required but we can update to the latest on chocolatey.
```suggestion
    (if "%python%"=="3.12" setx PYTHON_VERSION "3.12.6" && setx PATH "%PATH%;C:\Python312;C:\Python312\Scripts") & \
```